src/main/java/org/jabref/gui/preferences/ShowPreferencesAction.java

```diff
@@ -11,15 +13,25 @@ public class ShowPreferencesAction extends SimpleCommand {
 
     private final JabRefFrame jabRefFrame;
     private final TaskExecutor taskExecutor;
+    private Optional<Class<? extends PreferencesTab>> preferencesTabToSelectClass = Optional.empty();
 
     public ShowPreferencesAction(JabRefFrame jabRefFrame, TaskExecutor taskExecutor) {
         this.jabRefFrame = jabRefFrame;
         this.taskExecutor = taskExecutor;
     }
 
+    public ShowPreferencesAction(JabRefFrame jabRefFrame, TaskExecutor taskExecutor, Class<? extends PreferencesTab> preferencesTabToSelectClass) {
```

Comment:
Create an overloaded constructor without the preferencesTabToSelectClass argument

Reply:
Sorry, overlooked the other constructor 🤪